data.tf

```diff
@@ -0,0 +1,7 @@
+data "azurerm_client_config" "current" {}
+
+# Retrieve  ingest00-meta002-sbox kv
+data "azurerm_key_vault" "ingest00-meta002-sbox" {
+  name                = "ingest00-meta002-sbox"
+  resource_group_name = var.resource_group_name
+}
```

Comment:
would this always be `sbox` is the idea to make this module reusable across other environments, if so then ignore else would making this reusable be beneficial?